webcompat/form.py

```diff
@@ -210,4 +223,8 @@ def build_formdata(form_object):
             image_url=form_object.get('image_upload').get('url'))
     # Append "from webcompat.com" message to bottom (for GitHub issue viewers)
     body += u'\n\n{0}'.format(GITHUB_HELP)
-    return {'title': summary, 'body': body}
+    rv = {'title': summary, 'body': body}
+    extra_label = form_object.get('label', None)
+    if extra_label and extra_label in app.config['EXTRA_LABELS']:
+        rv.update({'labels': [extra_label]})
+    return rv
```

Comment:
Just a note for myself. 
`build_formdata` starts to be a long piece of code. Maybe we want in the future to break it in smaller components. Not sure.